mlflow/java/client/src/main/java/org/mlflow/tracking/MlflowClient.java

```diff
@@ -299,7 +299,7 @@ public void setTag(String runId, String key, String value) {
   }
 
   /**
-   * Delete a tag against the given run, given a String key.
+   * Delete a tag against the given run, given a String key. This is irreversible.
    * @param runId The ID of the run in which the tag exists to delete.
```

Comment:
The ID of the run in which the tag exists to delete. -> ID of the run that contains the tag to delete